packages/integrations/preact/src/index.ts

```diff
@@ -18,8 +18,26 @@ function getRenderer() {
 	};
 }
 
-function getViteConfiguration() {
+function getCompatRenderer(): AstroRenderer {
 	return {
+		name: '@astrojs/preact/compat',
+		clientEntrypoint: '@astrojs/preact/client.js',
+		serverEntrypoint: '@astrojs/preact/server.js',
```

Comment:
I wonder if these still work or if we need a separate entrypoint that references `preact/compat`?

Reply:
They certainly _used_ to be shared in the old compat integration, so I’ll try your `ssr.noExternal` suggestions first and see how I get on. Thanks Nate!